cmd/smd/routers.go

```diff
@@ -343,12 +343,12 @@ func (s *SmD) generateRoutes() Routes {
 			s.compEPBase,
 			s.doComponentEndpointsDeleteAll,
 		},
-		Route{
-			"doComponentEndpointQueryGet",
-			strings.ToUpper("Get"),
-			s.compEPBase + "/{xname}",
-			s.doComponentEndpointQueryGet,
-		},
+		//Route{
+		//	"doComponentEndpointQueryGet",
+		//	strings.ToUpper("Get"),
+		//	s.compEPBase + "/Query/{xname}",
+		//	s.doComponentEndpointQueryGet,
+		//},
```

Comment:
Why was this commented out?

Reply:
We found a potential bug where the same URI is being mapped to multiple handler functions (doComponentEndpointGet and doComponentEndpointQueryGet). Manny thought we should fix the URIs for future use but also comment these routes out since their API endpoints are commented out in the Swagger spec and seemingly aren't being used. @alvarez3-hpe: would it be better to not comment these routes out and just fix their URIs?

Reply:
The route wasn't supported to begin with and is even commented out in the swagger. Commenting it out here changes nothing, leaving it may cause problems later, and fixing the URL would add a new API that reports itself as unsupported. It should have been commented out to begin with.